### PR TITLE
src/ceph-crash.in: various enhancements and fixes

### PR DESCRIPTION
**Pull request by leseb**
See individual commits.

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by tchaikov — APPROVED** [review]

File: src/ceph-crash.in [hunk 1]
```diff
@@ -75,7 +75,7 @@ def scrape_path(path):
                     (metapath, p, os.path.join('posted/', p))
                 )
 
-def handler(signum, frame):
+def handler(signum):
```
Comment (dmick) [inline src/ceph-crash.in:82]: "used" and "unsed" in commit message should be "unused"

File: src/ceph-crash.in [hunk 2]
```diff
@@ -51,7 +51,7 @@ def post_crash(path):
         stderr = pr.communicate(input=f.read())
         rc = pr.wait()
         f.close()
-        if rc != 0:
+        if rc != 0 or stderr != "":
```
Comment (dmick) [inline src/ceph-crash.in:54]: just printing to stderr isn't an error, of course, so I don't know that calling this "failed" is really right, but it's a nit

**Review by dmick — APPROVED** [review]
Other than nits, lgtm

File: src/ceph-crash.in (hunk 2 above)
Comment (leseb) [reply src/ceph-crash.in:54]: Agreed, but as explained in https://github.com/ceph/ceph/pull/42055/commits/8a89e0105d466d9a7659f6d5bd26f4e38d0d2758, I feel like it's better than nothing.

**Comment by leseb** [comment]
jenkins test make check

**Comment by leseb** [comment]
https://tracker.ceph.com/issues/51591

**Pull request merged**